src/openscm_runner/adapters/ciceroscm_adapter/make_scenario_files.py

```diff
@@ -206,7 +220,7 @@ def write_scenario_data(self, scenarioframe, odir):
                     "[^a-zA-Z0-9_-]",
                     "",
                     _get_unique_index_values(scenarioframe, "scenario"),
-                )
+                )[:50]
```

Comment:
Why are all these `[:50]` hanging around?

Reply:
They kind of have to, or else everything breaks when names get too long. If I cut here manually, I know what I get and can compare... Otherwise fortran cuts, and I would need to keep track of that...

Reply:
Ah ok got it. Can you add a comment to that effect somewhere in the code please?